### PR TITLE
[14.0][FIX] cetmix_tower_server: Custom code can be executed on a single server at once

### PR DESCRIPTION
**Pull request by dessanhemrayev**
This commit addresses the issue of executing custom code in the wizard when multiple servers are selected.
The following changes have been made:

API/Backend:
- A `UserError` is raised if a user tries to run custom code for multiple servers in the wizard.

UI/UX:
- The "Run in wizard" button is hidden if multiple servers are selected.
- The "Rendered code" tab is not rendered if multiple servers are selected.

Task: 3663

## Release Notes

- **Documentation Updates**
	- Updated URLs in the README and usage documentation to point to the stable version `14.0`.
	- Clarified instructions regarding server creation and command execution, particularly for scenarios involving multiple servers.

- **User Interface Enhancements**
	- Adjusted visibility logic for the command execution wizard and the 'Run in wizard' button to improve user experience when selecting multiple servers.

- **New Tests**
	- Introduced a test for command execution behavior when multiple servers are selected, enhancing error handling validation.

### CODE REVIEW

File: cetmix_tower_server/readme/USAGE.md
Comment (ivs-cetmix) [inline cetmix_tower_server/readme/USAGE.md]: Button will be show only if single server is selected.

File: cetmix_tower_server/readme/USAGE.md
Comment (ivs-cetmix) [inline cetmix_tower_server/readme/USAGE.md]: ..will not be rendered..

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Some minor things to fix, otherwise looks good

File: cetmix_tower_server/readme/USAGE.md
Comment (dessanhemrayev) [reply cetmix_tower_server/readme/USAGE.md]: Thanks, I'll fix it

File: cetmix_tower_server/readme/USAGE.md
Comment (dessanhemrayev) [reply cetmix_tower_server/readme/USAGE.md]: Sorry, I'll fix it

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
`Run in wizard` button must be hidden if several servers are selected.
image

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml
Comment (ivs-cetmix) [inline cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml]: Domain syntax is incorrect:
`condition_1 AND  (condition_2 AND OR condition_3)`

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml
Comment (ivs-cetmix) [inline cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml]: This domain means the following:

```
show_servers=True
AND
(
    result!=False OR code=False OR command_id!=False
)
```

Which result in the following behaviour:
image

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

File: cetmix_tower_server/tests/test_command_wizard.py
Comment (ivs-cetmix) [reply cetmix_tower_server/tests/test_command_wizard.py]: New records are created within transactions which are rolled back after test is finished. So records don't remain in the DB after test has finished.

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py
Comment (ivs-cetmix) [inline cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py]: Why do we need an extra field here? This should be done with a domain on the form.

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py
Comment (dessanhemrayev) [reply cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py]: Thanks, understood
I'll fix it

**Comment by ivs-cetmix** [comment]
@dessanhemrayev please do NOT squash commits until you get all approvals

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py
Comment (ivs-cetmix) [reply cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py:192]: This is not relevant because the main method for command execution `def execute_command()` has `ensure_one` in it.

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py
Comment (ivs-cetmix) [reply cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py:192]: This is not relevant because the main method for command execution `def execute_command()` has `ensure_one` in it.

**Review by ivs-cetmix — APPROVED** [review]
Code review LGTM

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Comment by CetmixGitDrone** [comment]
What a great day to merge this nice PR. Let's do it!
Prepared branch [14.0-dev-ocabot-merge-pr-135-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-135-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 353fb9b4abd126672038563f6a5085fa95a84027. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**
